MAUI/Rotator/Getting-started.md

```diff
@@ -159,10 +185,12 @@ namespace Rotator
 {% endtabs %}
 
 ![Rotator Items](images/RotatorItems.png)
+
+N> Ensure that the images mentioned in the code snippets are located in the **Resources** folder of your sample project.
```

Comment:
Move the note section above the code snippet

Reply:
Addressed the comments.

Reply:
why rotator image note added in 2 places? before code snippet and after the code snippet